mockredis/client.py

```diff
@@ -1509,6 +1513,8 @@ def mock_redis_client(**kwargs):
     """
     return MockRedis()
 
+mock_redis_client.from_url = mock_redis_client
```

Comment:
Looks good to me. I will release this after finishing other PRs. Thank you for your PR.